src/Sdk/DTLogging/Logging/ValueEncoders.cs

```diff
@@ -60,6 +60,20 @@ public static String XmlDataEscape(String value)
             return SecurityElement.Escape(value);
         }
 
+        public static String TrimDoubelQuoteDataEscape(String value)
+        {
+            var trimmed = string.Empty;
+            if (!string.IsNullOrEmpty(value) &&
+                value.StartsWith('"') &&
+                value.EndsWith('"') &&
+                value.Length > 2)
```

Comment:
do we need min length for the string?

Reply:
I would. If it's less than N then it's not a very good secret... maybe 5 or 6?

Reply:
So 5 or 6, plus 2 more for the surrounding quotes.